### PR TITLE
feat: Bedrock docs for AI Proxy and AI Proxy Advanced

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

 

DOCU-3863
### Testing instructions

Preview link: 
- https://deploy-preview-7913--kongdocs.netlify.app/hub/kong-inc/ai-proxy/how-to/machine-learning-platform-integration-guides/bedrock/
- https://deploy-preview-7913--kongdocs.netlify.app/hub/kong-inc/ai-proxy-advanced/how-to/machine-learning-platform-integrations/bedrock/

### Checklist 

- [x] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/_hub/kong-inc/ai-proxy-advanced/how-to/machine-learning-platform-integrations/bedrock.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+1. Create a route that maps to the service you defined:
+```sh
+curl -i -X POST http://localhost:8001/routes \
+    --data "paths[]=/bedrock" \
+    --data "service.id=$(curl -s http://localhost:8001/services/bedrock-service | jq -r '.id')"
+```
+
+1. Use the Kong Admin API to configure the AI Proxy Advanced Plugin to route requests to Amazon Bedrock:
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/ai-proxy-advanced/how-to/machine-learning-platform-integrations/bedrock.md:31]: For full disclosure, I used AI to help me fix my formatting for `targets` (tried to do it by hand, got some errors with {} etc.). But I also visually checked the output to make sure all the fields were there and then ran this example and the plugin successfully installed.

**Comment by cloudjumpercat** [comment]
Tested both workflows top-to-bottom and both worked.

File: app/_hub/kong-inc/ai-proxy-advanced/how-to/machine-learning-platform-integrations/bedrock.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                "aws_region": "us-east-1"
+              }
+            }
+          }
+        }
+      ]
+    }
+  }'
+```
+Be sure to replace the following:
```
Comment (Guaris) [inline app/_hub/kong-inc/ai-proxy-advanced/how-to/machine-learning-platform-integrations/bedrock.md:61]:
```suggestion
Be sure to replace the following values:
```

File: app/_hub/kong-inc/ai-proxy/how-to/machine-learning-platform-integration-guides/bedrock.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+          "bedrock": {
+            "aws_region": "us-east-1"
+          }
+        }
+      }
+    }
+  }'
+```
+
+Be sure to replace the following:
```
Comment (Guaris) [inline app/_hub/kong-inc/ai-proxy/how-to/machine-learning-platform-integration-guides/bedrock.md:58]:
```suggestion
Be sure to replace the following values:
```

**Review by Guaris — APPROVED** [review]

**Pull request merged**